crates/prql-compiler/src/semantic/std.prql

```diff
@@ -195,3 +195,8 @@ let upper = column -> <text> internal std.upper
 ## File-reading functions, primarily for DuckDB
 let read_parquet = source<text> -> <relation> internal std.read_parquet
 let read_csv = source<text> -> <relation> internal std.read_csv
+
+
+## PRQL compiler functions
+
+let prql_version = -> <text> internal prql_version
```

Comment:
@Hulxv I tried removing the `std.`, and it worked. 

I think that's the distinction between some of these that require going later in the pipeline vs. those that don't. But it's not that clear, I think we can use this effort to improve a lot how we can add functions...